src/dtgtk/thumbnail.c

```diff
@@ -226,10 +224,26 @@ static void _image_get_infos(dt_thumbnail_t *thumb)
   const dt_image_t *img = dt_image_cache_get(darktable.image_cache, thumb->imgid, 'r');
   if(img)
   {
-    thumb->has_localcopy = (img->flags & DT_IMAGE_LOCAL_COPY);
     thumb->rating = img->flags & DT_IMAGE_REJECTED
       ? DT_VIEW_REJECT
       : (img->flags & DT_VIEW_RATINGS_MASK);
+  }
```

Comment:
Is there any reason to close the `if(img)` condition here (and reopen it after) ?
I think it would even break the `dt_image_cache_release` just after (in case of `OVERLAYS_NONE`)

In fact I even think that current code in master is not fully right : in the case we fail to get the image cache, it actually "reset" the `thumb->rating` to 0. I would have said that in this case, we have to let everything untouched...
So for our case that would means : 
- removing the `thumb->rating = 0` line
- let all your change inside the `if(img)` condition
- add another check after that condition to exit if no overlay

What would you think of suvh a change ? @TurboGit : I would also like your opinion as I fail to trigger a case where we don't get the image cache...

Thanks !